src/utils/navigation.ts

```diff
@@ -62,7 +62,7 @@ export const NAVIGATION: Navigation = {
   [config.UPDATE_REVIEW_STATEMENT_URL]: {
     currentPage: config.UPDATE_REVIEW_STATEMENT_PAGE,
     previousPage: () => config.UPDATE_NO_CHANGE_REGISTRABLE_BENEFICIAL_OWNER_URL,
-    nextPage: []
+    nextPage: [config.WHO_IS_MAKING_UPDATE_PAGE, config.UPDATE_NO_CHANGE_BENEFICIAL_OWNER_STATEMENTS_PAGE]
```

Comment:
Should we add a 'todo' here referencing that UAR 585 will change this to go to payments page?

Reply:
That was a hangover from a merge. Changed. Cheers,